proxy/envoy/ingress.go

```diff
@@ -276,6 +276,9 @@ func buildIngressRoute(ingress *proxyconfig.IngressRule,
 	if err != nil {
 		return nil, "", err
 	}
+	if servicePort.Protocol != model.ProtocolHTTP {
```

Comment:
Ingress is only for HTTP traffic in-cluster. It may expose HTTPS on the front side, but that's irrelevant here.